docs/static/releasenotes.asciidoc

```diff
@@ -29,6 +30,170 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-8-0]]
+=== Logstash 7.8.0 Release Notes
+
+==== New features and improvements
+
+===== Expanded JDK ecosystem 
+
+We have improved test scripts infrastructure to accept a specific Java
+Development Kit (JDK) present in the host. The changes include exporting the
+path to a JDK home into an environment variable named BUILD_JAVA_HOME and its
+use by shell scripts to launch test and integration scripts. This work and other
+JDK14 fixes pave the way for Logstash to support both AdoptOpenJDK 11 and 14 in
+the near future . https://github.com/elastic/logstash/pull/11786[#11786],
+https://github.com/elastic/logstash/pull/11839[#11839],
+https://github.com/elastic/logstash/pull/11935[#11935]
```

Comment:
I'd like to rework this section to be more user-focused. Please help me understand the user benefits.

Reply:
for now there isn't a user facing benefit, this is more to let users know we're expanding how logstash is tested, and that will create a comfortable baseline for us to be more flexible and reactive to supporting new JDK and deprecating old ones.

Reply:
IMHO, we should have consolidated both "Expanded JDK Ecosystem" and "Expanded JDK Platform Support" and completely reword this as a user level benefit, further expanding that our testing infrastructure now covers this. Also, I think we should not longer link pull requests in this section, but link them on a further section below with the rest of the changelog.